src/main/java/models/Game.java

```diff
@@ -82,13 +91,13 @@ public void customDeal(Player p, int c){
     public void WhoWins(){
         active = false;
         if((user.sum>dealer.sum || dealer.sum>22)&&user.sum<22) {
-            winner = "user";
+            winner = "USER";
             user.dough=user.dough+(2*bet);
         }
         else if((user.sum<dealer.sum || user.sum>22)&& dealer.sum<22)
-            winner = "dealer";
+            winner = "DEALER";
         else{
```

Comment:
except there are multiple cases where a tie can occur (both when the score is the same and when both players bust)